src/Files.Uwp/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml

```diff
@@ -590,7 +590,7 @@
                         Header="{x:Bind Header, Mode=OneWay}"
                         IconSource="{x:Bind IconSource, Mode=OneWay}"
                         Style="{StaticResource Local.TabViewItem}"
-                        ToolTipService.ToolTip="{x:Null}" />
+                        ToolTipService.ToolTip="{Binding HoverDisplayText}" />
```

Comment:
```suggestion
                        ToolTipService.ToolTip="{x:Bind HoverDisplayText, Mode=OneWay}" />
```
Can you try this?

Reply:
It works perfectly! I'm closing this side of the conversation, thanks for your help! :)